pkg/controller/cluster_controller.go

```diff
@@ -18,16 +18,18 @@ package controller
 
 import (
 	"context"
+	"time"
+
 	api "github.com/cockroachdb/cockroach-operator/api/v1alpha1"
 	"github.com/cockroachdb/cockroach-operator/pkg/actor"
 	"github.com/cockroachdb/cockroach-operator/pkg/resource"
 	"github.com/go-logr/logr"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	policy "k8s.io/api/policy/v1beta1"
```

Comment:
it would be helpful to either rename the above includes to drop `v1` or use the same scheme.

Reply:
I'm not making those changes in this PR, since it does not deal with this PR, if you want to open an issue that would be amazing.  I think those changes need to happen across multiple files.